### PR TITLE
Update manage-families.md

### PR DESCRIPTION
**Pull request by LaureBro**
Reworked the article and added some more information.

### CODE REVIEW

File: content/md/first-steps/manage-families.md [hunk 1]
```diff
@@ -1,112 +1,102 @@
 ---
 id: manage-your-families
 themes: catalog-settings
-title: Manage your **families**
+title: Manage your **families** and **variant families**
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: Sorry, you cannot put several part of the title in bold. It's not working currently.

File: content/md/first-steps/manage-families.md [hunk 2]
```diff
@@ -1,112 +1,102 @@
...
+
+Once a family is created, you can configure your family:
 *   Add/remove attributes from the family
 *   Edit these attributes’ completeness
-*   To select the attribute to be used as «label» for the products in the product grid (column "label") and in their product form
-*   To select the attribute to be used as «the main picture» for the products in the product grid (column "image") and in their product form header
 
 :::info
-The attributes available to be used as «the main picture» are attributes not localizable and not scopable, with the types `image` or  `assets collection` (since 2.1).
+You must be granted a specific permission to be able to create a family.
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: `role permission` + add a link to the `Build your user roles` article. (https://help.akeneo.com/articles/build-your-user-roles.html)

File: content/md/first-steps/manage-families.md [hunk 3]
```diff
@@ -1,112 +1,102 @@
...
-The family is now created.
-
-Once a family is created, you can later edit those family information:
-*   The family’s labels for the locales enabled for the catalog
+Your new family is now created. The family's `Properties` tab is displayed.     
+You can define some additional properties for your family:      
+*   An `attribute used as label (required)`: the attribute that will be used as «Label» for the products belonging to this family. The Label will appear in the product grid ("Label") column and it will also be used in the product form's header.
+**Only text and identifier attributes can be used as a product's Label.**
+*   An `attribute used as the main picture`: the attribute that will be used as «the main picture» for the products belonging to this family in the product grid (column "image") and in the product form's header.
+You can select an `image` or  `assets collection` (since 2.1) attribute types. Attributes that localizable and/or scopable cannot be used as the main product picture.
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: `that are localizable`

File: content/md/first-steps/manage-families.md [hunk 4]
```diff
@@ -1,112 +1,102 @@
...
-1.  Under the `Properties` tab, change the labels
-1.  Click on `Save`
-
 ## Manage attributes in a family
-Attributes can be shared amongst families, but an attribute can be only added once to a family.
-
-You have several ways of adding an attribute, a group of attribute or several attributes to a family.
+Attributes can be used in one or several families, but **an attribute can only be added once to a family**.         
+    
+You can easily add attributes to or remove them from a family using the user interface or through imports.
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: Can we put a link to the Import article on the `import` word?

File: content/md/first-steps/manage-families.md [hunk 5]
```diff
@@ -117,35 +107,111 @@ On the attribute’s family page:
...
 1.  Click on `Save`
 
 :::warning
-The attribute removed will be taken in account for the completeness computing. Completeness will be updated next time it is executed.
+Once changes are saved, the PIM will automatically run a completeness calculation to update your product information.
+:::
+
+## Mass edit families
+
+Like for products you can mass edit families and add several attributes at once for several families, and through the same action define if they are required of not for the selected channels.
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: `required or not`

File: content/md/first-steps/manage-families.md [hunk 6]
```diff
@@ -117,35 +107,111 @@ On the attribute’s family page:
...
+1.  The mass edit window will be displayed
+1.  Select the operation `Set attributes requirements`
+1.  Click on `Next`
+![image](../img/Settings_FamiliesBulkActions2.png)
+1.  Add the relevant attributes by attributes or by attribute groups
+1.  Set the attributes required for the product completeness of the channels
+1.  Click on `Next`
+1.  Confirm the action with the `Confirm` button to add attributes to the selected families     
+
+The attributes have been added to families.
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: `to the families`

File: content/md/first-steps/manage-families.md [hunk 7]
```diff
@@ -117,35 +107,111 @@ On the attribute’s family page:
...
+
+## Check the family history
+
+Changes made on a family are recorded and visible under the `History` tab.      
+
+For each change, a version of the family is created. The following information is tracked down for each version:
+- The person or system that generated changes
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: `generated the changes`

File: content/md/first-steps/manage-families.md [hunk 8]
```diff
@@ -117,35 +107,111 @@ On the attribute’s family page:
...
+
+You have two ways of removing a family.
+Through the family form:
+1. Go to `Settings/Families` to list the families
+1. Click on the family to delete
+1. Click on `...` for other actions and then `Delete` on the top right corner
+  ![image](../img/Settings_FamilyDelete.png)
+1. Confirm the action via the validation message
+
+Through the Delete shortcut in the grid:
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: Delete between `

File: content/md/first-steps/manage-families.md [hunk 9]
```diff
@@ -117,35 +107,111 @@ On the attribute’s family page:
...
+  ![image](../img/Settings_Families_DeleteHover.png)
+1. Select the family you want to remove from the family datagrid
+1. Place your mouse over the line of family in the grid, the button delete is prompted (trash can icon)
+1. Click on the trash can icon
+1. Confirm the action via the validation message
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: `in` instead of `via`?

File: content/md/first-steps/manage-families.md [hunk 10]
```diff
@@ -154,15 +220,17 @@ The family variants are managed in a dedicated `Variants` tab in `Settings/Famil
...
-- The **variant attributes for the level 1** in the 2nd column are the attributes which vary according to the variant axes of the level 1
+- The **variant attributes for the level 1** in the 2nd column are the attributes which will vary according to the variant axes of the level 1
 - If the family variant has 2 variant levels, **the variant attributes for the level 2** in the 3rd column are the attributes which vary according to the variant axes of the level 2 (and level 1)
 
-In our example for clothing which vary on 2 levels by color and then by size, the common attributes are the name, the collection, the description, the brand, etc. The attributes varying by color (variant level 1) are the composition and the pictures and the attributes varying by color and size (variant level 2) are the EAN, the SKU and the weight.
+In our clothing example which vary on 2 levels by color and then by size, the common attributes are: the name, the collection, the description, the brand, etc. The attributes that vary by color (variant level 1) are the composition, the pictures and the attributes that differ by color and size (variant level 2) are the EAN, the SKU and the weight.
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: `varies on 2 levels`

File: content/md/first-steps/manage-families.md [hunk 11]
```diff
@@ -154,15 +220,17 @@ The family variants are managed in a dedicated `Variants` tab in `Settings/Famil
...
 
-## Create a family variant
-1.  In `Settings/Families`, choose a family and open the `Variants` tab
+## Create a new family variant
+
+You can create family variant for all your families.
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: `create family variants`

File: content/md/first-steps/manage-families.md [hunk 12]
```diff
@@ -239,68 +294,14 @@ A family variant can be deleted if there is no [product model](/articles/what-ab
...
-  ![image](../img/Settings_Families_DeleteHover.png)
-1. Select the family you want to remove from the family datagrid
-1. Place your mouse over the line of family in the grid, the button delete is prompted (trash can icon)
-1. Click on the trash can icon
-1. Confirm the action via the validation message
+1.  In `Settings/Families`, choose a family and open the `Variants` tab
+1.  Place your mouse over the line of the family variant you want to remove in the grid, the button delete is prompted (trash can icon)
+![Display a family variant](../img/Settings_Families_Variant_Delete.png)
+1.  Click on the trash can icon, a confirmation opens
+1.  Click on `Delete` to confirm the action via the confirmation message
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: `in` instead of `via`

File: content/md/first-steps/manage-families.md [hunk 13]
```diff
@@ -1,112 +1,102 @@
...
 themes: catalog-settings
-title: Manage your **families**
+title: Manage your **families and variant families**
 popular: false
 ee-only: false
 related: what-is-a-family
 ---
 
-# List families
+:::info
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: Maybe to remove, as we already have a link to this article at the end in the `Related articles` section?

Comment (LaureBro) [reply content/md/first-steps/manage-families.md]: Hum, do you want me to remove the List Families part? or put back the text we had before and remove the info part? (I think we should remove the 1st part: List families)

**Pull request merged**